homeassistant/components/xiaomi_miio/light.py

```diff
@@ -790,9 +786,11 @@ def hs_color(self) -> tuple:
         return self._hs_color
 
     @property
-    def supported_features(self):
-        """Return the supported features."""
-        return SUPPORT_BRIGHTNESS | SUPPORT_COLOR | SUPPORT_COLOR_TEMP
+    def color_mode(self):
+        """Return the color mode of the light."""
+        if self.hs_color:
```

Comment:
This should be the same behavior as without the PR.
I don't think it's really correct though; the light will be stuck in `hs` mode once a color is set.

Does the `miio` library offer any way to get the color mode for a `PhilipsMoonlight`?

Reply:
The library does not provide that at the moment and I'm not sure if it can be derived from the data returned by the device. Maybe @syssi knows if that's even possible?

Reply:
@rytilahti @syssi If the library offers no way to get the color mode I suggest to merge this PR, it should not result in any changed behavior

Reply:
I think this is fine as an (intermediate) solution.